enterprise/templates/enterprise/templatetags/alert_messages.html

```diff
@@ -2,8 +2,10 @@
 
 <div id="messages">
   {% for message in messages %}
-    <div class="alert {{ message.tags }}" aria-live="polite">
-      {% fa_icon message.level_tag %} {{ message }}
-    </div>
+    {% autoescape off %}
+      <div class="alert {{ message.tags }}" aria-live="polite">
+        {% fa_icon message.level_tag %} {{ message }}
+      </div>
+    {% endautoescape %}
```

Comment:
I think all you need to do here is mark the message variable as safe instead of turning autoescape off for the block:
```
<div class="alert {{ message.tags }}" aria-live="polite">
  {% fa_icon message.level_tag %} {{ message|safe }}
</div>
```